### PR TITLE
[test] Attempt to import from registry without the CA config map existing

### PR DESCRIPTION
**Pull request by awels**
Verify that the pod remains pending for a while until we create the config map and then the import succeeds.

Signed-off-by: Alexander Wels <awels@redhat.com>

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: tests/datavolume_test.go [hunk 1]
```diff
@@ -811,6 +813,50 @@ var _ = Describe("[vendor:cnv-qe@redhat.com][level:component]DataVolume tests",
...
+				// Sleep one second and try again.
+				time.Sleep(time.Second)
+			}
+
+			By("Creating the config map")
+			_, err = utils.CopyRegistryCertConfigMapDestName(f.K8sClient, f.Namespace.Name, f.CdiInstallNs, cmName)
+			Expect(err).To(BeNil())
+
+			By(fmt.Sprintf("waiting for datavolume to match phase %s", string(cdiv1.Succeeded)))
+			utils.WaitForDataVolumePhase(f.CdiClient, f.Namespace.Name, cdiv1.Succeeded, dataVolume.Name)
```
Comment (maya-r) [inline tests/datavolume_test.go:857]: Missing check of return value?

also: I assume the cleanup is handled by the namespace choice, is that correct?

Comment (awels) [reply tests/datavolume_test.go:857]: Yeah part of the framework is deleting the namespace at the end, which will in turn delete the data volume/pvc/pods that are in that namespace. Let me add the check.

File: tests/datavolume_test.go [hunk 2]
```diff
@@ -811,6 +813,50 @@ var _ = Describe("[vendor:cnv-qe@redhat.com][level:component]DataVolume tests",
...
+			By("verifying pvc was created")
+			Eventually(func() bool {
+				// TODO: fix this to use the mechanism to find the correct PVC once we decouple the DV and PVC names
+				pvc, _ = f.K8sClient.CoreV1().PersistentVolumeClaims(dataVolume.Namespace).Get(dataVolume.Name, metav1.GetOptions{})
+				return pvc != nil && pvc.Name != ""
+			}, timeout, pollingInterval).Should(BeTrue())
+
+			By("Verifying the POD remains pending for 30 seconds")
+			podName := naming.GetResourceName(common.ImporterPodName, pvc.Name)
+			for i := 0; i < 30; i++ {
```
Comment (tomob) [inline tests/datavolume_test.go:836]: Why not use `Consistently` here?

Comment (awels) [reply tests/datavolume_test.go:836]: Because I didn't know Consistently existed. I will modify as it much clearer intend.

Comment (tomob) [reply tests/datavolume_test.go]: That's a fair reason :wink: :+1:

**Comment by tomob** [comment]
LGTM

**Pull request merged**